src/elements/common/feature-checking/__tests__/FeatureChecking.test.js

```diff
@@ -76,6 +76,23 @@ describe('FeatureFlag', () => {
         expect(enabledFn).toHaveBeenCalledWith(foo);
         expect(disabledFn).toHaveBeenCalled();
     });
+
+    test('"not" prop inverts flag', () => {
+        const enabledFn = jest.fn(() => null);
+        const disabledFn = jest.fn(() => null);
+        const foo = { otherProp: 'foo' };
+        mount(
+            <FeatureProvider
+                features={{
+                    foo,
+                }}
+            >
+                <FeatureFlag disabled={disabledFn} enabled={enabledFn} not feature="foo" />
```

Comment:
I find this a bit confusing, personally. Why is inverting the flag necessary? If we have `enabled` and `disabled` it seems like we cover both conditions.

Reply:
+2, adding a `not` makes it rather confusing.

Reply:
And if we need a use case where you have to render something based on if something is not ON, can we just have a component that does it. `<InvertedFeatureFlag>` or a better name, though that's only a little bit less confusing.

Reply:
Here's an example: https://github.com/box/box-ui-elements/compare/master...alexkrolick:feat/tasks/new-api#diff-a844e542fc9fc2f6103b82dd5410fb75R69

I was passing `enabled` and `disabled` as inline functions, but got a request to refactor as a class method on the previous PR. That would require rewriting the function component as a class component. Adding this prop allows the use of children instead of functions while negating the feature.

Reply:
Added test for children prop https://github.com/box/box-ui-elements/pull/834/files#diff-a73cef8be86971e9d516ffd86e661da4R79